### DISCUSSION TITLE
大佬，希望您能增加一个可以自己调颜色的功能。

### DISCUSSION THREAD

**Discussion by NoclubOIRzXx** [Planshit/Tai#62] [Ideas]
大佬，希望您能增加一个可以自己调颜色的功能。就是在分类页面调，在统计页面就能清楚看。

也可以加一个悬浮窗。
可以显示在任务栏也可以悬浮在窗口顶端，记录目前最大化窗口的已运行时间，关闭窗口停止计时，打开窗口重新计时并显示在虚浮窗上。最小化暂停计时，打开后继续计时。

显示效果这样挺好
![image](https://user-images.githubusercontent.com/40497676/191405369-c52c0d8e-cc13-4417-80f0-4e7d9952e125.png)
这个显示在我的屏幕左下角，系统为Win11

**Comment by noberumotto** [comment]
收到。但是短期内可能不会有更新，因为最近实在太忙了

**Comment by noberumotto** [comment]
分类颜色功能已经加入，如果还在用Tai的话记得更新~

**Reply by NoclubOIRzXx** [reply]
欧耶，大佬牛皮

**Reply by noberumotto** [reply]
欢迎继续反馈问题或者提建议哦~

**Reply by NoclubOIRzXx** [reply]
![image](https://user-images.githubusercontent.com/40497676/195657078-1090873b-afc1-46ad-aade-8f28cf48c5ca.png)
大佬，UI 有点小问题。

能加一个选项吗？显示模式自己选择：按时长总和显示或者目前原有的显示模式

**Reply by noberumotto** [reply]
不太明白你的意思，目前是按每个分类的时长总和呀
